src/libmongoc/src/mongoc/mongoc-cursor-cmd.c

```diff
@@ -49,12 +49,8 @@ _getmore_type (mongoc_cursor_t *cursor)
    wire_version = server_stream->sd->max_wire_version;
    mongoc_server_stream_cleanup (server_stream);
 
-   if (
-      /* Server version 5.1 and newer do not support OP_GETMORE. */
-      wire_version > WIRE_VERSION_5_0 ||
-      /* Fallback to legacy OP_GETMORE wire protocol messages if exhaust cursor
-         requested. */
-      !_mongoc_cursor_get_opt_bool (cursor, MONGOC_CURSOR_EXHAUST)) {
+   /* TODO (CDRIVER-4722) always GETMORE_CMD when server >= 4.2 */
```

Comment:
```suggestion
   // CDRIVER-4722: always GETMORE_CMD once WIRE_VERSION_MIN >= WIRE_VERSION_4_2.
```
Suggest avoiding use of `TODO` for comments that correspond to JIRA tickets (the "todo" status is tracked/documented by the ticket).

Suggestion also applies to other TODO comments referring to CDRIVER-4722.